### PR TITLE
Add Publications

### PR DESCRIPTION
**Pull request by dlebauer**
adds publications section

### CODE REVIEW

File: content/publication/schnaufer-2020/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "An open, scalable, and flexible framework for automated aerial measurement of field experiments"
+date: 2020-05-28
+publishDate: 2020-08-05T21:26:26.413799Z
+authors: ["Christophe Schnaufer", "Julian L Pistorius", "David S LeBauer"]
+publication_types: ["1"]
+abstract: "Unoccupied areal vehicles (UAVs or drones) are increasingly used in field research.
```
Comment (Chris-Schnaufer) [inline content/publication/schnaufer-2020/index.md]:
```suggestion
abstract: "Unoccupied aerial vehicles (UAVs or drones) are increasingly used in field research.
```

**Review by Chris-Schnaufer — CHANGES_REQUESTED** [review]

**Review by KristinaRiemer — COMMENTED** [review]
This is really neat! I especially like the ability to filter publications by their type. 

I have thoughts on a couple aesthetic things. First, I'm not a fan of this weird alert thing at the top of the section and think it should be removed. 

Screen Shot 2020-08-07 at 12 56 22 PM

Second, I'm also not a fan of the little paper bullet points here, and I don't there needs to be any sort of bullet for the list of pubs. 

Screen Shot 2020-08-07 at 12 58 09 PM

I'm interested to hear others' opinions on these two matters. I'm happy to be the person to implement whatever changes we decide upon.

**Comment by Chris-Schnaufer** [comment]
> ... not a fan of this weird alert thing ...

I'm not a fan either until I need them. If it could be changed into just the link `filtering publications` (or some text like that) I think it'd be better. For now it's OK as it is.

> ... little paper bullet points ...

Given the way the text is organized with the white space around each publication, I think the paper bullet points are not needed. If the blocks of text end up having white space within them, then the little paper bullet points should be there.

**Comment by KristinaRiemer** [comment]
At the bottom of the publications section, there's this link "See All Publications" that leads to the same page as the alert's "filtering publications" link. 

Screen Shot 2020-08-10 at 6 54 46 AM

So those two links are redundant. We could move the "See All Publications" link to the top of the section and rename it something like "Filter and view all publications" or something?

**Review by KristinaRiemer — APPROVED** [review]

**Pull request merged**